python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,103 @@
+Estimation of a quantile upper bound by Wilks' method
+-----------------------------------------------------
 
-Let us denote
-:math:`\underline{Y} = h\left( \vect{X},\vect{d} \right) = \left( Y^1,\ldots,Y^{n_Y} \right)`,
-where :math:`\vect{X}= \left( X^1,\ldots,X^{n_X} \right)` is a random
-vector, and :math:`\vect{d}` a deterministic vector. We seek here to
-evaluate, using the probability distribution of the random vector
-:math:`\vect{X}`, the :math:`\alpha`-quantile :math:`q_{Y^i}(\alpha)` of
-:math:`Y^i`, where :math:`\alpha \in (0, 1)`:
+We consider a random variable :math:`X` of dimension 1 and its unknown quantile :math:`x_{\alpha}` of order :math:`\alpha`.
+We seek to evaluate an upper bound of :math:`x_{\alpha}` with a confidence greater or equal to :math:`\beta`, using a given order statistics.
+
+Let :math:`(X_1, \dots, X_\sampleSize)` be some independent and identically distributed (i.i.d.) variables according to :math:`X`.
+
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th maximum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum. We have:
 
 .. math::
 
-   \begin{aligned}
-       \Prob{ Y^i \leq q_{Y^i}(\alpha)} = \alpha
-     \end{aligned}
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)}
+
 
-If we have a sample
-:math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of :math:`N`
-independent samples of the random vector :math:`\vect{X}`,
-:math:`q_{Y^i}(\alpha)` can be estimated as follows:
+Smallest rank for an upper bound to the quantile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  the sample :math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of
-   vector :math:`\vect{X}` is first transformed to a sample
-   :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` of the variable
-   :math:`Y^i`, using :math:`\underline{y} = h(\vect{x}_i,\vect{d})`,
+Let :math:`(x_1, \dots, x_\sampleSize)` be an i.i.d. sample of size :math:`\sampleSize` of
+the random variable :math:`X`.
+Given a quantile level :math:`\alpha \in [0,1]`, a confidence level
+:math:`\beta \in [0,1]`, and a sample size :math:`\sampleSize`, we seek for the smallest
+rank :math:`k \in \llbracket 1, \sampleSize \rrbracket` such that:
 
--  the sample :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` is then placed
-   in ascending order, which gives the sample
-   :math:`\left\{ y^{(1)},\ldots,y^{(N)} \right\}`,
+.. math::
+    :label: EqOrderStat
 
--  this empirical estimation of the quantile is then calculated by the
-   formula:
+    \Prob{x_{\alpha} \leq X_{(k)}} \geq \beta
 
-   .. math::
+The order statistics :math:`X_{(k)}` follows the distribution which probability density
+function and cumulative distribution function
+are:
```

Comment:
The order of the statements is reversed. It is easier to read it as follows in my opinion:
```suggestion
The probability density and cumulative distribution functions of the order 
statistics :math:`X_{(k)}` are:
```